basket/news/tasks.py

```diff
@@ -328,19 +322,6 @@ def fxa_direct_update_contact(fxa_id, data):
 
     Ignore if contact with FxA ID can't be found
     """
-    try:
-        sfdc.contact.update(f"FxA_Id__c/{fxa_id}", data)
-    except sfapi.SalesforceMalformedRequest as e:
-        if e.content[0]["errorCode"] == "REQUIRED_FIELD_MISSING":
-            # couldn't find the fxa_id and tried to create a record but doesn't
-            # have the required data to do so. We can drop this one.
-            return
-        else:
-            # otherwise it's something else and we should potentially retry
-            raise
-    except SFDCDisabled:
-        pass
-
     basket_data = from_sfdc(data)
```

Comment:
Can we lose `from_sfdc` now, too, or is that going in a later pass?

Reply:
I'm going to work on that next. Basically bypass the `from_sfdc` by creating the `data` in the right format to begin with. Then we can remove it.

Reply:
This is now removed.